README.md

```diff
@@ -4,4 +4,64 @@ SPDX-FileCopyrightText: 2021 Benedict Harcourt <ben.harcourt@harcourtprogramming
 SPDX-License-Identifier: BSD-2-Clause
 -->
 
-# Kitteh's Bot Framework
+# MewBot
+
+MewBot is an automation framework intended to simplify building cross-platform
+text/chat-bots.
+The design is intended to be modular with configuration separated from code,
+allowing users and developers to build out custom logic and behaviours with
+minimal coding experience.
+
+### Status
+
+> :warning: This project is still in the very early stages. Some basic bots can be built
+> and run, but we currently consider all parts of the framework to be unstable.
+
+### Packages and Modules
+
+| Package                   | Modules                             | Description                                                                                                                                |
+|---------------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `mewbot-core`             | `mewbot.core`                       | Base interfaces for all modules                                                                                                            |
+| `mewbot-dev`              | `mewbot.component`, `mewbot.api.v1` | Development libraries + component registry system. This is the package that all 3rd party libraries should depend on for their interfaces. |
+| `mewbot-runner`           | `mewbot.loader`, `mewbot.bot`       | Tools to load a bot, and run that bot.                                                                                                     |
+| `mewbot-[discord/twitch]` | `mewbot.io.[discord/twitch]`        | The bindings to connect MewBot to a given service.                                                                                         |
+
+![module dependency graph](./mewbot.svg)
```

Comment:
Shout .bot depend on .discord or have I misunderstood what .bot does?

Reply:
`.bot` should not need to have a direct dependency on anything in the API/IO side -- it cares only for the interfaces and not the implementations. The `.loader` module will use `importlib` to find modules as components are loaded (from YAML).